cvat-ui/src/components/annotation-page/top-bar/right-group.tsx

```diff
@@ -2,27 +2,40 @@
 //
 // SPDX-License-Identifier: MIT
 
-import React from 'react';
-import { Col } from 'antd/lib/grid';
 import Icon from '@ant-design/icons';
-import Select from 'antd/lib/select';
+import { changeAnnotationsFilters, fetchAnnotationsAsync } from 'actions/annotation-actions';
 import Button from 'antd/lib/button';
-
-import { DimensionType, Workspace } from 'reducers/interfaces';
-import { InfoIcon, FullscreenIcon } from 'icons';
+import { Col } from 'antd/lib/grid';
+import Select from 'antd/lib/select';
+import { FilterIcon, FullscreenIcon, InfoIcon } from 'icons';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { CombinedState, DimensionType, Workspace } from 'reducers/interfaces';
 
 interface Props {
     workspace: Workspace;
     showStatistics(): void;
+    showFilters(): void;
     changeWorkspace(workspace: Workspace): void;
     jobInstance: any;
 }
 
 function RightGroup(props: Props): JSX.Element {
     const {
-        showStatistics, changeWorkspace, workspace, jobInstance,
+        showFilters, showStatistics, changeWorkspace, workspace, jobInstance,
     } = props;
 
+    const filters = useSelector((state: CombinedState) => state.annotation.annotations.filters);
+    const dispatch = useDispatch();
+    const filtersClickHandler = (e: React.MouseEvent<HTMLElement, MouseEvent>): void => {
+        if (e.altKey) {
+            dispatch(changeAnnotationsFilters([]));
+            dispatch(fetchAnnotationsAsync());
+        } else {
```

Comment:
Guess it can be better to add popup with helper for such case

Reply:
@ActiveChooN 
This was an additional quick way to clear the filters. We already have particular button on the panel. I believe in this case we can just remove alt-click listener.